pyproject.toml

```
@@ -21,6 +25,9 @@ classifiers = [
 ]
 readme = "README.md"
```

explicitly add license info: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license
```suggestion
readme = "README.md"
license = "MIT"
```

---

also maybe any useful `[project.urls]`?
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls

---

The license entry in "classifiers" could have remained, I think. According to the guide linked above it helps with searches on PyPi, e.g.: https://pypi.org/search/?c=License+%3A%3A+OSI+Approved+%3A%3A+MIT+License

---

It should be either `cliassifiers` or `license` not both. Otherwise, it'll fail to build package.